components/telemetry-operator/config/crd/bases/telemetry.kyma-project.io_logparsers.yaml

```diff
@@ -0,0 +1,79 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.0
+  creationTimestamp: null
+  name: logparsers.telemetry.kyma-project.io
+spec:
+  group: telemetry.kyma-project.io
+  names:
+    kind: LogParser
+    listKind: LogParserList
+    plural: logparsers
+    singular: logparser
+  scope: Cluster
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .status.conditions[-1].type
+      name: Status
+      type: string
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: LogParser is the Schema for the logparsers API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: LogParserSpec defines the desired state of LogParser
+            properties:
+              parser:
+                description: Parser is an example field of LogParser. Edit logparser_types.go
```

Comment:
Fix this description by setting proper comments on the type (here and check all others)